src/openrpc/localization.json

```diff
@@ -274,7 +274,105 @@
           }
         }            
       ]   
-    } 
+    },
+    {
+      "name": "addAdditionalInfo",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-manages": [
+            "xrn:firebolt:capability:localization:additional-info"
+          ]
+        }
+      ],
+      "summary": "Add any platform-specific localization information in key/value pair",
+      "params": [
+        {
+          "name": "key",
+          "summary": "Key to add additionalInfo",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "value",
+          "summary": "Value to be set for additionalInfo",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        }
+      ],
+      "result": {
+        "name": "success",
```

Comment:
Probably shouldn't name the parameter 'success'.

I usually name it 'result' in cases like this.

Reply:
Makes sense... will change the parameter name for both the methods.